apps/ntag/arduinoNtag/ntagDefs.h

```diff
@@ -0,0 +1,21 @@
+/* SWI defines for ntag_main script */
+#ifndef NTAGDEFS_H
+#define NTAGDEFS_H
+
+#include "ntageepromadapter.h"
+
+#define HARDI2C
+
+
+/* SWI mangOH Yellow has the 2K Ntag */
+static Ntag ntag(Ntag::NTAG_I2C_2K,2,5);
```

Comment:
What is going on here?  Will this define static variables in every compilation unit that includes this file?  What are you trying to achieve?